concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountAddress.java

```diff
@@ -32,6 +38,25 @@ public static AccountAddress from(String address) {
         return AccountAddress.from(addressBytes);
     }
 
+    public AccountAddress newAlias(int alias) {
+        if (alias < 0) {
+            throw new NumberFormatException("Alias must be non negative");
+        }
+        if (alias > ALIAS_MAX_VALUE) {
+            alias = alias % ALIAS_MAX_VALUE;
+        }
+        val buffer = ByteBuffer.allocate(Integer.BYTES);
+        buffer.putInt(alias);
+        val newAlias = this.bytes.clone();
+        System.arraycopy(buffer.array(), 1, newAlias, ACCOUNT_ADDRESS_PREFIX_SIZE, 3);
+        return AccountAddress.from(newAlias);
+    }
+
+    public boolean isAliasOf(AccountAddress other) {
+        return Arrays.equals(Arrays.copyOfRange(this.bytes, 0, ACCOUNT_ADDRESS_PREFIX_SIZE),
```

Comment:
Interesting, you have to copy arrays to compare parts of them? No slicing?

Reply:
I don't think there is an *easy* way with this java version at least.?